examples/cdn-example/index.js

```diff
@@ -0,0 +1,42 @@
+const liff = window.liff;
+const liffMockPlugin = window.liffMock;
+
+liff.use(new liffMockPlugin());
+
+const main = () => {
+  document.getElementById('init_btn').onclick = onInitClick;
+  document.getElementById('login_btn').onclick = login;
+  document.getElementById('get_profile_btn').onclick = getProfile;
+};
+
+const onInitClick = async () => {
+  try {
+    await liff.init({
+      liffId: 'xxx',
+      mock: true,
+    });
+    console.log('liff.init done');
+  } catch (e) {
+    console.error('liff.init error', e);
+  }
+};
+
+const login = async () => {
+  try {
+    await liff.login();
+  } catch (e) {
+    console.log('liff.login done');
```

Comment:
Move to try block?

Reply:
Good Catch! DONE 92c17971259fd98740451878eb7e83cfa163d08b